RockLib.Configuration/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 4.0.0-alpha.1 - Not Yet Released
+
+#### Changed
+- Removed .NET Core 3.1 as a TFM and added .NET 8. Supported targets are now .NET 4.8, .NET 6, and .NET 8.
+- Removed RockLib.Immutable as a dependency.
```

Comment:
Might want to add a little more detail about removing it and adding functionality to this library.